package.json

```diff
@@ -123,6 +134,9 @@
 		"progress": "2.0.3",
 		"react": "16.8.4",
 		"react-dom": "16.8.4",
+		"react-native": "0.59.9",
+		"react-native-recyclerview-list": "0.3.4",
+		"react-native-safe-area": "0.5.1",
```

Comment:
I'm a little bit concerned about the added dependencies here. Since most of the time, these are not needed in this repository. Maybe like "chromium" install in e2e tests we could find ways to lazy-install those while running the tests.

Curious if there's any precedent in other projects and how people organize mobile and web versions together? Do they use separate folders with separate package.json? do they mix everything?...

Reply:
> Maybe like "chromium" install in e2e tests we could find ways to lazy-install those while running the tests

I'm curious about this because I see Chromium being downloaded if I run `npm install`.

I'd argue that if most of the time we don't need these, it's because this is just the first step in bringing the platforms closer. One of the longer term goals is that anyone can test their changes on mobile as easily as they can do with the web with the docker setup.

Reply:
> I'm curious about this because I see Chromium being downloaded if I run npm install.

I think there's an envionment variable we set in CI to do so. (I thought it was also in local but apparently not) `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD`

> I'd argue that if most of the time we don't need these, it's because this is just the first step in bringing the platforms closer. One of the longer term goals is that anyone can test their changes on mobile as easily as they can do with the web with the docker setup.

Fair enough, we can try and improve over time.

Reply:
I removed a couple of dependencies that are not really needed:
- `react-native-recyclerview-list`
- `react-native-safe-area`

I added mocks for those instead.